<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MAP_GUI_TitleBar</class>
 <widget class="QWidget" name="MAP_GUI_TitleBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>30</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0,0">
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QPushButton" name="m_TitleBarIconBtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/icon/icon/square-o.png</normaloff>:/new/icon/icon/square-o.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_TitleBarIconBtn2">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/icon/icon/square-o.png</normaloff>:/new/icon/icon/square-o.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_TitleBarIconBtn3">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/new/icon/icon/square-o.png</normaloff>:/new/icon/icon/square-o.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_TitleBarTitleLabel">
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>标题栏</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_TitleBarMinimizePtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/Icon/icon/minimize-black.png</normaloff>:/Icon/icon/minimize-black.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>12</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_TitleBarMaximizePtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/Icon/icon/maxmize-normal-black.png</normaloff>:/Icon/icon/maxmize-normal-black.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>12</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_TitleBarClosePtn">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>25</height>
      </size>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resource.qrc">
       <normaloff>:/Icon/icon/close-black.png</normaloff>:/Icon/icon/close-black.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>12</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
